helpers/tistream.py

```diff
@@ -0,0 +1,21 @@
+from StringIO import StringIO
+
+class TiedStream(StringIO):
+
+    def __init__(self, buf='', tied=None):
+        self.tied = tied
+        #print('TiedStream init')
```

Comment:
Please remove all commented our `print()` statements. We have debugging for this purpose - use it.